modules/flowable-cmmn-converter/src/test/java/org/flowable/test/cmmn/converter/CasePageTaskCmmnXmlConverterTest.java

```diff
@@ -49,64 +48,66 @@ public void convertModelToXML() throws Exception {
         // Plan items definitions
         List<PlanItemDefinition> planItemDefinitions = planModel.getPlanItemDefinitions();
-        assertEquals(2, planItemDefinitions.size());
-        assertEquals(2, planModel.findPlanItemDefinitionsOfType(CasePageTask.class, false).size());
+        assertThat(planItemDefinitions).hasSize(2);
+        assertThat(planModel.findPlanItemDefinitionsOfType(CasePageTask.class, false)).hasSize(2);
         
         // Plan items
         List<PlanItem> planItems = planModel.getPlanItems();
-        assertEquals(2, planItems.size());
+        assertThat(planItems).hasSize(2);
         
         PlanItem planItemTaskA = cmmnModel.findPlanItem("planItemTaskA");
         PlanItemDefinition planItemDefinition = planItemTaskA.getPlanItemDefinition();
-        assertEquals(0, planItemTaskA.getEntryCriteria().size());
-        assertTrue(planItemDefinition instanceof CasePageTask);
+        assertThat(planItemTaskA.getEntryCriteria()).isEmpty();
+        assertThat(planItemDefinition).isInstanceOf(CasePageTask.class);
 
         CasePageTask taskA = (CasePageTask) planItemDefinition;
-        assertEquals(CasePageTask.TYPE, taskA.getType());
-        assertEquals("A", taskA.getName());
-        assertEquals("testKey", taskA.getFormKey());
-        assertEquals("Label 1", taskA.getLabel());
-        assertEquals("Icon 1", taskA.getIcon());
-        assertEquals("johndoe", taskA.getAssignee());
-        assertEquals("janedoe", taskA.getOwner());
-        assertEquals(2, taskA.getCandidateUsers().size());
-        assertEquals("johndoe", taskA.getCandidateUsers().get(0));
-        assertEquals("janedoe", taskA.getCandidateUsers().get(1));
-        assertEquals(2, taskA.getCandidateGroups().size());
-        assertEquals("sales", taskA.getCandidateGroups().get(0));
-        assertEquals("management", taskA.getCandidateGroups().get(1));
-        assertNotNull(planItemTaskA.getItemControl());
-        assertNotNull(planItemTaskA.getItemControl().getParentCompletionRule());
-        assertEquals(ParentCompletionRule.IGNORE, planItemTaskA.getItemControl().getParentCompletionRule().getType());
+        assertThat(taskA.getType()).isEqualTo(CasePageTask.TYPE);
+        assertThat(taskA.getName()).isEqualTo("A");
+        assertThat(taskA.getFormKey()).isEqualTo("testKey");
+        assertThat(taskA.getLabel()).isEqualTo("Label 1");
+        assertThat(taskA.getIcon()).isEqualTo("Icon 1");
+        assertThat(taskA.getAssignee()).isEqualTo("johndoe");
+        assertThat(taskA.getOwner()).isEqualTo("janedoe");
+        assertThat(taskA.getCandidateUsers()).hasSize(2);
+        assertThat(taskA.getCandidateUsers().get(0)).isEqualTo("johndoe");
+        assertThat(taskA.getCandidateUsers().get(1)).isEqualTo("janedoe");
+        assertThat(taskA.getCandidateGroups()).hasSize(2);
+        assertThat(taskA.getCandidateGroups().get(0)).isEqualTo("sales");
+        assertThat(taskA.getCandidateGroups().get(1)).isEqualTo("management");
```

Comment:
Can be replaced with `assertThat(taskA.getCandidateGroups()).containsExactly("sales", "management");`

Reply:
Thanks; done.